calculator.c

```diff
@@ -0,0 +1,53 @@
+//Olivier Gabison - A simple calculator.
+
+#import <stdio.h>
```

Comment:
Not python - you meant include! It set off a warning. Aside from this, solid job.